roles/ceph-osd/tasks/activate_osds.yml

```diff
@@ -41,6 +41,23 @@
 - include: osd_fragment.yml
   when: crush_location
 
+- name: check if selinux is enabled
+  command: getenforce
+  register: sestatus
+  changed_when: false
+
+- name: set selinux to permissive
```

Comment:
could you please use the selinux module instead? http://docs.ansible.com/ansible/selinux_module.html